test/Website/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM microsoft/dotnet:1.0-runtime
```

Comment:
(1.1.0-runtime is out!)

For base image tag, I'm not sure what's the difference between `1.0-runtime` and `1.0.3-runtime`, I suppose the former will be updated whenever there is a new 1.0.x release. This makes our dockerfile less predictable as the base image version is not immutable.

I would lock the version of the base image down to the exact version of the dotnet image, e.g. 1.0.3-runtime